README.md

```diff
@@ -84,14 +84,6 @@ Go to the `docker` directory and build the containers (this may take several min
 docker-compose --profile prod build
 ```
 
-### Initializing databases
-
-The databases need to be initialized, again from the `docker` directory:
-```
-docker-compose --profile prod up munimap-postgis munimap-postgis-mapbender
-```
-When the message `database system is ready to accept connections` is displayed the initialization is complete, and the containers can be stopped with `ctrl+c`.
-
```

Comment:
This is removed because in a follow-up PR the app container will start only if the databases are ready.